frontend/src/audio/index.js

```diff
@@ -0,0 +1,9 @@
+import {router} from 'san-router';
```

Comment:
There should be any need for this file now. The index.js was used in San. With the Vue framework, we should need to implement this file anymore.

Reply:
ya, i will remove it